pkg/endpoint/endpoint.go

```diff
@@ -335,8 +335,9 @@ type Endpoint struct {
 	// other resources
 	controllers controller.Manager
 
-	// proxy redirects to remove later during the build
-	proxiesToRemove map[string]bool
+	// proxiesToRemove is the set of IDs of proxy redirects to remove later
+	// during the build.
+	proxiesToRemove map[string]struct{}
```

Comment:
What is the `struct{}`?

Reply:
This is the value type of maps classically used when one doesn't care about the value. Using a bool gave the illusion that the boolean value mattered, which wasn't the case.

Reply:
@aanm this allows for uniqueness (and thus a set) - the `struct{}` is a placeholder

Reply:
Always learn something new!